snmp/datadog_checks/snmp/data/default_profiles/_dell-rac.yaml

```
@@ -6,6 +6,8 @@ metrics:
       OID: 1.3.6.1.4.1.674.10892.5.4.200.10
       name: systemStateTable
     symbols:
+      - name: systemStatePowerSupply
```

```suggestion
      - name: systemStateTable
```

Should be `systemStateTable` instead since we are using symbols from `systemStateTable` ?